doc/source/serve/performance.md

```diff
@@ -42,29 +100,34 @@ Given these symptoms, there are several ways to fix the issue.
 Make sure you are using the right hardware and resources.
 Are you using GPUs (`ray_actor_options={“num_gpus”: 1}`)? Are you using one or more cores (`ray_actor_options={“num_cpus”: 2}`) and setting [`OMP_NUM_THREADS`](serve-omp-num-threads) to increase the performance of your deep learning framework?
 
-### `async` methods
+### Using `async` methods
 
 Are you using `async def` in your callable? If you are using `asyncio` and
 hitting the same queuing issue mentioned above, you might want to increase
 `max_concurrent_queries`. Serve sets a low number (100) by default so the client gets
-proper backpressure. You can increase the value in the Deployment decorator; e.g.
+proper backpressure. You can increase the value in the deployment decorator; e.g.
 `@serve.deployment(max_concurrent_queries=1000)`.
 
-### Batching
+### Batching requests
 
-If your deployment can process a batch at a time at a sublinear latency
-(for example, if it takes 1ms to process 1 query and 5ms to process 10 of them)
-then batching is your best approach. Check out the [batching guide](serve-batching) to
-make your deployment accept batches (especially for GPU-based ML inference). You might want to tune `max_batch_size` and `batch_wait_timeout` in the `@serve.batch` decorator to maximize the benefits:
+If your deployment can process batches at a sublinear latency
+(meaning, for example, that it takes say 1ms to process 1 query and 5ms to process 10 of them)
+then batching is your best approach. Check out the [batching guide](serve-batching) and
+refactor your deployment to accept batches (especially for GPU-based ML inference). You might want to tune `max_batch_size` and `batch_wait_timeout` in the `@serve.batch` decorator to maximize the benefits:
 
 - `max_batch_size` specifies how big the batch should be. Generally,
   we recommend choosing the largest batch size your function can handle
-  AND the performance improvement is no longer sublinear. Take a dummy
-  example: suppose it takes 1ms to process 1 query, 5ms to process 10 queries,
+  without losing the sublinear performance improvement.
+  For example, suppose it takes 1ms to process 1 query, 5ms to process 10 queries,
   and 6ms to process 11 queries. Here you should set the batch size to to 10
```

Comment:
batch size to 10

Reply:
Nice catch, I removed the redundant `to`.